codecov.yml

```diff
@@ -36,5 +42,10 @@ flags:
     paths:
     - "src/sentry/**/*.py"
     carryforward: true
+  acceptance:
+    paths:
+    - "static/app/"
```

Comment:
do frontend and acceptance changes potentially both come from `/static/app` ? I don't think this will impact anything codecov related, just pointing out that the same path is pointed to by two different flags in the event that that's a typo. 

I notice L#48 is similar to L#43, so I assume this is intentional.

Reply:
@eliatcodecov I do not know the implication of these `paths` values. I've read the documentation but there's no clarification as to what their purpose is. Is it used anywhere? or is it just for the reader's sake?

Since `acceptance` would be a joint of the frontend and backend flags I assumed the paths should be a joint.

Anyone that could enlighten me as to the implications of getting the paths, right?

Reply:
@armenzg We will limit the paths we carryforward to the paths specified in the `paths` array. If you do not include, all files from the reports uploaded under that specific flag will be carried forward. 

Generally, unless there's some reason you have files that appear in your coverage reports that you do not want to carryforward, I would probably leave the `paths` array empty. We'll get the docs updated to better reflect this behavior.